packages/jtex/src/jtex.ts

```diff
@@ -145,6 +150,7 @@ class JTex {
       errorLogFn: errorLogger(this.session),
       warningLogFn: warningLogger(this.session),
     };
+    if (bibliography) frontmatter.bibliography = bibliography;
     const validatedDoc = validateTemplateDoc(frontmatter, templateYml?.doc || [], options, opts);
```

Comment:
Can you just check if we should be destructuring frontmatter before changing the bib?
ie.
```
const validatedDoc = validateTemplateDoc({...frontmatter, bibliography: bibliography ?? frontmatter.bibliography }, templateYml?.doc || [], options, opts);
 ```